### DISCUSSION TITLE
👋🏻 Hi, My name is Saurav Maheshkar and I'm excited to be here. 😊

### DISCUSSION THREAD

**Discussion by SauravMaheshkar** [OpenMined/courses#77] [Introduce Yourself]
Hi :wave:,
I’m a senior high school student from New Delhi, India and I want to pursue deep learning in my higher education. I spent the last 2 years studying Machine Learning having completed multiple specialisations from deeplearning.ai, Google, IBM, Imperial College London, the OpenMined Udacity Course (ofc 😊) and many other courses (Have a look at my website for the complete list). Before that, I’ve worked as a Front End Developer with 6 months of internship experience in a leading educational startup. I also have a tech firm which I started with a couple of friends and we take industry projects for App and Web development. I’ve also worked a lot with IoT and Robotics having won multiple hackathon’s and representing my projects in the National Science Fair consecutively for 2 years. Two of my projects on Fake News Classification and Regression Analysis of Primary Biliary Cirrhosis are currently being mentored by Intel under the AI 4 Youth program.

I hope to pursue my Bachelors ( Hopefully with integrated masters ) from the United Kingdom. I’ve applied and already gotten an offer from the University of Birmingham, hoping to get into the University of Edinburgh. 🤞 

I’m currently learning about **Attention**, **Self-Supervised Representation Learning**, **Graphical Models** and **Medical Imaging** (Prognosis, some regression analysis as well).

Here, are my socials, if you want to connect: [Twitter](https://twitter.com/MaheshkarSaurav), [LinkedIn](https://www.linkedin.com/in/saurav-maheshkar-643382187/), [Github](https://github.com/SauravMaheshkar), [Kaggle](https://www.kaggle.com/sauravmaheshkar) and my [Personal Website](https://sauravvmaheshkar.gitbook.io/saurav-maheshkar/)

I've worked a lot with Tensorflow, Pytorch, JAX Based ecosystems (Trax, Flax, Objax, ...). I'm almost done with Cynthia Dwork's book The Algorithmic Foundations of Differential Privacy. I've been reading some academic papers in Differential Privacy for about 3 months and am interested to learn how to deploy these systems in production.

**Comment by iamtrask** [comment]
Hi Saurav! Welcome to the course! Congratulations on all your specializations! You've really been taking advantage of lots of great content coming out. Best of luck in your application to Edinburgh - they do have a fantastic program. If you end up moving do let me know and I can introduce you to some folks in the area.

Congratulations on pushign through Algorithmic Foundations of DP as well - that's a seriously non-trivial read. You might consider joining the Differential Privacy team if you're interested in meeting other folks interested in the topic (we're going to be doing a pretty big push on web app development and DP in the future as well - something you might be good at given your background!)

**Reply by SauravMaheshkar** [reply]
Hey, @iamtrask thanks for the reply. You made my day 🤯 😄 . Happy belated birthday as well. 🎂 

Funny you said about the DP team, I actually just applied yesterday for the Openmined Research Engineer position (DP + SplitNN + Healthcare). Let's see what happens.

Congratulations on the course release. 😊 and thanks for the offer to connect me with the folks at Edinburgh. Fingers crossed 🤞 I get in.
